src/scenic/simulators/webots/model.scenic

```diff
@@ -119,6 +153,89 @@ class Ground(WebotsObject):
             y += dy
         return tuple(heights)
 
+    @staticmethod
+    @distributionFunction
+    def shapeFromHeights(heights, width, length, gridSizeX, gridSizeY, baseThickness):
+        heights = [row[::-1] for row in heights[::-1]]
+
+        triangles = []
+
+        ## Vertices
+        dx, dy = width / (gridSizeX - 1), length / (gridSizeY - 1)
+        base_x, base_y = -width / 2, length / 2
+
+        heightmap_vertices = tuple(tuple((base_x+ix*dx, base_y-iy*dy, heights[iy][ix]) for ix in range(gridSizeX)) for iy in range(gridSizeY))
+
+        raw_vertices = [v for row in heightmap_vertices for v in row]
+        top_vertices = list(dict.fromkeys(raw_vertices))
```

Comment:
What is this line supposed to do? As far as I can tell `top_vertices` is just `raw_vertices` but possibly in a different order.

Reply:
I think I initially wanted to remove duplicate vertices but there shouldn't be any here... I'll investigate.